Practice9_Tests/Calculator_Tests.cs

```diff
@@ -1,8 +1,80 @@
-﻿using System;
+﻿using NUnit.Framework;
+using Practice_9;
 
 namespace Practice9_Tests
 {
+    [TestFixture]
     public class Calculator_Tests
     {
+        [Test]
+        public void TestAddPass()
+        {
+            double x = 2;
+            double y = 2;
+            Calculator calc = new Calculator(x, y);
+            Assert.AreEqual(4, calc.Add(x, y));
+        }
+
+        [Test]
+        public void TestAddFail()
```

Comment:
usually, tests are not expected to be failed.
in general, they all should pass.

In addition, the tests are usually named more specific, e.g. "AddTwoValues" in this case.